src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1553768043.inc.php

```diff
@@ -0,0 +1,29 @@
+<h1>Build #1553768043</h1>
+<h2>Date: 2019-03-28</h2>
+<div class="changelog">
+    - Reset field class and fix translations for cms_media.width
+</div>
+<?php
+
+$fieldId = TCMSLogChange::GetTableFieldId(TCMSLogChange::GetTableId('cms_media'), 'width');
```

Comment:
@ Backwards incompatibility:
You could add an update message (INFO) here that the "width" field was restored.

And if somebody really needs the current state they can revert this update.

That would make it easier for 99% and give the potential 1% using the old media manager the chance to know and correct.

Reply:
done